OTKit/otkit-typography-styles/token.yml

```diff
@@ -0,0 +1,69 @@
+props:
+  # each grouping of styles is namespaced via kebab case.
+  # a style must have 4 values (not enforced via tooling, yet).
+  # "size": the font size
+  # "color": the font color
+  # "weight": the font weight
+  # "line-height": the line height
+
+  h1-size:
+    value: "{!font-size-xlarge}"
+  h1-color:
+    value: "{!color-gray-primary}"
+  h1-weight:
+    value: "{!font-weight-bold}"
+  h1-line-height:
+    value: "{!line-height-xlarge}"
+
+  # note: h2's have 3 different variants.
+  # on light backgrounds, they can be bold or not;
+  # on dark backgrounds, they must be bold.
+  h2-bold-light-size:
+    value: "{!font-size-large}"
+  h2-bold-light-color:
+    value: "{!color-gray-primary}"
+  h2-bold-light-weight:
+    value: "{!font-weight-bold}"
+  h2-bold-light-line-height:
+    value: "{!line-height-large}"
+  h2-light-size:
+    value: "{!font-size-large}"
+  h2-light-color:
+    value: "{!color-gray-primary}"
+  h2-light-weight:
+    value: "{!font-weight-normal}"
+  h2-light-line-height:
+    value: "{!line-height-large}"
+  h2-dark-size:
+    value: "{!font-size-large}"
+  h2-dark-color:
+    value: "#FFFFFF"
+  h2-weight:
+    value: "{!font-weight-bold}"
+  h2-line-height:
+    value: "{!line-height-large}"
+
+  h3-size:
+    value: "{!font-size-medium}"
+  h3-color:
+    value: "{!color-gray-primary}"
+  h3-weight:
+    value: "{!font-weight-normal}"
```

Comment:
Now that you mention this, I noted you [used](https://github.com/opentable/design-tokens/blob/master/OTKit/aliases.yml#L81) 'normal' and 'bold' what that does it means? I think that they should have an actual weight value I guess @lastquestion ?